packages/admin-ui/src/lib/core/src/shared/components/facet-value-chip/facet-value-chip.component.ts

```diff
@@ -13,4 +13,9 @@ export class FacetValueChipComponent {
     @Input() removable = true;
     @Input() displayFacetName = true;
     @Output() remove = new EventEmitter<void>();
+
+    get formattedTitle(): string {
+        const facetCode = this.facetValue.facet.code ? `(${this.facetValue.facet.code}) ` : '';
```

Comment:
What's the need for this line? Do you expect that the `facet.code` may sometimes be falsy?

Reply:
I expect the code to be empty (I've updated the code to reflect this) if it's not requested in the graphql request
https://github.com/vendure-ecommerce/vendure/pull/3175/commits/b499e03717164d0a0556f5edcb97aaebf0a80f4f